### PR TITLE
Create a yaml for joint weights 

### PR DESCRIPTION
**Pull request by jatinmayekar**
- Create a joint weights yaml to set the scaling factor for the distance metrics of the manipulators
- This pull request will help in checking the pull request for the moveit repository which adds a feature for scaling the distance metrics for the manipulators https://github.com/ros-planning/moveit/pull/1758

### CODE REVIEW

File: config/joint_weights.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# joint_weights.yaml allows to set the the scaling factors for the distance metrics of the manipulators 
+# Specific joint properties can be changed with the keys [weight]
+
+
+joint_weights:
```
Comment (mamoll) [inline config/joint_weights.yaml:5]: Why not simply write the list like so:

```
joint_weights:
   panda_joint1: 0.5
   panda_joint2: 0.5
   ...
```

Comment (jatinmayekar) [reply config/joint_weights.yaml:5]: I have made it similar to the joint_limits.yaml i.e, to use the same variable for different joints. In the robot_model_loader.cpp it becomes easier to assign the same variable to distance factors of different joints. So it's independent of joint names as we already have an object pointing to the different joints.

**Review by rhaschke — CHANGES_REQUESTED** [review]
I agree to @mamoll: `joint_limits.yaml` defines several variables per joint. However, here only a single weight is defined. Hence, please change to the syntax proposed by Mark.

**Comment by v4hn** [comment]
This requests depends on the outcome of https://github.com/ros-planning/moveit/issues/1446

**Comment by GabrielKoenig** [comment]
Please take a look at my thoughs in https://github.com/ros-planning/moveit/issues/1446#issuecomment-559934674 before merging....

**Pull request closed without merging**